index.bs

```diff
@@ -422,14 +425,22 @@ it shouldn't be used in practice
 as it is easy to filter out the added noise.
 
 
-<h4 id="informing-user">Keeping the user informed about API use</h4>
+<h4 id="inform-user">Keep the user informed about API use</h4>
 
 User agents may choose to keep the user informed
 about current and past use of the API.
 
 Note: this does not imply keeping a log of the actual [=sensor readings=]
 which would have issues of its own.
 
+<h4 id="input-elements">Input Elements</h4>
+
+When an <{input}> element, whose [=browsing context container=]'s [=node document=]'s
+[=origin=] is not [=same origin-domain=] with its [=parent browsing context=]'s document's
+[=origin=], is [=focused=], the user agent should [=stop the sensor=], or alternatively,
```

Comment:
"the user agent should [=stop the sensor=] […] [=limit maximum polling frequency=]" --> "the user agent should [=stop the sensor|stop=] all sensors running in its [=parent browsing context=]."